### PR TITLE
MVP guidelines for Slider

### PR DESCRIPTION
**Pull request by kpoulincoveo**
###Proposed Changes
revised all guidelines

###Potential Breaking Changes
N\A

###Acceptance Criteria
This I don't know 👇
-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Review by aiheon — APPROVED** [review]

**Review by sallainCoveo — APPROVED** [review]

**Review by gdostie — APPROVED** [review]

**Review by toofff — APPROVED** [review]
LGTM

File: packages/website/docs/Slider.md [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
+
+Use positive and negative symbols as prefixes when the range allows negative values.
+
+## Feedback and Validation
 
-[Toggle](#/components/FlatSelect)
+A slider always has a default value. Consider the most frequently set value as default first. If none particularly stands out, consider using a neutral value or the median. Note that the default value should stand on a marker as much as possible.
 
-[Radio Button](#/components/Radio)
+If a **numeric** value between two markers can be chosen, display the selected value above the handle or next to the slider so users can easily refer to what they have selected.
```
Comment (jlevesquecoveo) [inline packages/website/docs/Slider.md:31]:
```suggestion
If users can select a numeric value between two markers, display this value above the handle or next to the slider. This allows users to refer to what they selected.
```

File: packages/website/docs/Slider.md [hunk 2]
```diff
@@ -1,33 +1,39 @@
...
 
-[Dropdown List](#/components/DropdownSearch)
+---
+
+## Related Components
 
-[Slide Toggle](#/styles/form-controls/slide-toggle)
+If your use case doesn't match the guidelines above, consider using one of the following components instead:
```
Comment (jlevesquecoveo) [inline packages/website/docs/Slider.md:37]:
```suggestion
If setting a precise value is important, consider using a [numeric input](#/form/NumericInput) instead.
```

File: packages/website/docs/Slider.md [hunk 3]
```diff
@@ -1,33 +1,39 @@
...
 
-[Checkbox](#/components/Checkbox)
+-   [Numeric input](#/form/NumericInput) when setting a precise value is important.
```
Comment (jlevesquecoveo) [inline packages/website/docs/Slider.md:39]: If you accepted my previous comment, delete this.

**Review by jlevesquecoveo — DISMISSED** [review]

File: packages/website/docs/Slider.md (hunk 2 above)
Comment (aiheon) [reply packages/website/docs/Slider.md:37]: @kpoulincoveo

File: packages/website/docs/Slider.md (hunk 3 above)
Comment (aiheon) [reply packages/website/docs/Slider.md:39]: @kpoulincoveo

**Review by gdostie — APPROVED** [review]

File: packages/website/docs/Slider.md (hunk 3 above)
Comment (kpoulincoveo) [reply packages/website/docs/Slider.md]: Accepted @aiheon ready to merge

**Pull request merged**